<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Splash</class>
 <widget class="QWidget" name="Splash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
    <widget class="QLabel" name="logo_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
    <widget class="QLabel" name="app_name_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
    <widget class="QLabel" name="version_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress_bar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="status_label">
     <property name="text">
      <string>Loading...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
